### PR TITLE
Feature/write basic unit tests

### PR DESCRIPTION
**Pull request by Josephhaefling**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 
## What is the current behavior?
- Previously there were no unit tests.
## What is the new behavior?
- Currently, all unit tests are passing. We may need a few more.
## Does this introduce a breaking change?
- [ ] Yes
- [x] No
## Other information
n/a

### CODE REVIEW

File: cypress/integration/Components/App.test.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ describe('ChooseVoiceFrom', () => {
     cy.get('.header').contains('Gift of Gab').should('be.visible')
   })
 
-  it('should render the sub-category page', () => {
-    cy.get('.sub-category-container').should('be.visible')
+  it('should render the main page', () => {
```
Comment (jeremiahblackol) [inline cypress/integration/Components/App.test.js:15]: seems simple enough

**Review by jeremiahblackol — APPROVED** [review]
dope getting this set up

File: cypress/integration/Components/ChooseVoiceForm.test.js [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+            cy.get('label').contains('Sample Phrase').should('be.visible')
+            cy.get('input').should('be.visible')
+            cy.get('label').contains('Choose your voice').should('be.visible')
+            cy.get('select').should('be.visible')
+            cy.get('label').contains('Speed').should('be.visible')
+            cy.get('h5').contains('0').should('be.visible')
+            cy.get('button').contains('+').should('be.visible')
+            cy.get('button').contains('-').should('be.visible')
+            cy.get('button').contains('Play').should('be.visible')
+            cy.get('button').contains('Save').should('be.visible')
```
Comment (jdvolk) [inline cypress/integration/Components/ChooseVoiceForm.test.js:29]: Looks great

File: cypress/integration/Components/Navigation.test.js [hunk 3]
```diff
@@ -3,7 +3,11 @@ describe('ChooseVoiceFrom', () => {
...
     })
 
-    it('should be true', () => {
-        expect(true).to.equal(true)
+    it('should render the home button', () => {
+        cy.get('Button').contains('Home').should('be.visible')
+        cy.get('Button').contains('Options').should('be.visible')
+        cy.get('Button').contains('Help').should('be.visible')
+        cy.get('Button').contains('Bathroom').should('be.visible')
+
```
Comment (jdvolk) [inline cypress/integration/Components/Navigation.test.js:11]: nice unit tests!

**Review by jdvolk — APPROVED** [review]

**Pull request merged**